apps/studio/components/interfaces/Settings/API/JWTSettings.tsx

```diff
@@ -226,6 +224,11 @@ const JWTSettings = () => {
                                 <p className="text-sm opacity-50">
                                   A random secret will be created, or you can create your own.
                                 </p>
+                                <p className="text-sm opacity-50">
+                                  This will invalidate all existing API keys, including your{' '}
+                                  <code className="text-xs">service_role</code> and{' '}
+                                  <code className="text-xs">anon</code> keys.
+                                </p>
```

Comment:
"This will invalidate your anon and service_role API keys, and log out all signed in users via Supabase Auth."